tensorboard/data/server/masked_crc.rs

```diff
@@ -30,7 +30,13 @@ pub struct MaskedCrc(pub u32);
 // Implement `Debug` manually to use zero-padded hex output.
```

Comment:
Is this comment still relevant for `Debug`, now that it writes `Masked({})` without using the `:#010x` padding option?

Reply:
Yes, I think so. The point of the comment is to explain why we don’t
just write `#[derive(Debug)]`. This patch changes the implementation,
but not the behavior. If we derived `Debug`, the output would still be
`MaskedCrc(4660)` rather than `MaskedCrc(0x1234)`, so the comment is
still as helpful as it was before.

In fact, I think it’s a bit more helpful than before, since before the
`:#010x?` was at least immediately following in the code. Now, if you
don’t look at the `Display` implementation, it’d be reasonable to ask,
“this just wraps the `Display` in the struct name; at that point, why
not just derive `Debug`?”—which the comment answers.

Reply:
I guess the tests answer it, too. :-)